provider/pkg/provider/repository.go

```diff
@@ -97,9 +96,9 @@ func NewRegistryGeoReplication(ctx *pulumi.Context,
 	component.Registry = registryResource
 	component.LoginServer = registryResource.LoginServer
 	replication, err := containerregistry.NewReplication(ctx, "replicationResource", &containerregistry.ReplicationArgs{
-		Location:          pulumi.String(acrReplicaLocationParam),
+		Location:          pulumi.String(resourceGroupVar.Location),
```

Comment:
I'm not sure that this is right. Won't this mean we create a replicator from a location to itself. 

If we are going to stop using `acrReplicaLocationParam` we should stop taking it as an input. It exists in `ReplicationName` but no where else.